README.md

```diff
@@ -21,7 +21,7 @@
 - DeviceSize
 - FormattedTextField
 - Forms
-- HairlineView
+- [HairlineView](Pod/Classes/HairlineView/README.md) - Simple horizontal or vertical hairline view
```

Comment:
```suggestion
- [HairlineView](Pod/Classes/HairlineView/README.md) - horizontal or vertical hairline view
```
I think using "simple" here boxes us in. For example, we might add the ability to do dotted lines or other stroke patterns later.

Reply:
I didn't intend "Simple" to be conflated with "plain" or "inflexible". Perhaps "Easy to use" would be more clear?

Reply:
Don't think it needs to be called out. I wouldn't read "horizontal or vertical hairline view" and think "I'd use that, but only if it were easier to use!" 😉